Analysis/backgrounds_zgwg.tex

```diff
@@ -231,6 +238,7 @@ \subsection{Data-driven Control Regions}
   }
   \label{fig:tf_syst}
 \end{figure}
+\PH{Why are there two sets of uncertainties, there is just one going from W=>Z, the other direction is 100\% correlated, is this just that one set but applied in each sample?. Reading further the labels and text I think are wrong since you show the Z=>W ratio later}
```

Comment:
it's split by which process is contributing, simply because having them all on one plot would be too messy. left is uncertainties from Zgamma xsec, right is from Wgamma xsec. updated text to better explain